packages/terra-menu/src/terra-dev-site/doc/menu/Menu.1.doc.mdx

```diff
@@ -0,0 +1,57 @@
+import { Badge } from 'terra-menu/package.json?dev-site-package';
+
+import MenuPropsTable from 'terra-menu/src/Menu?dev-site-props-table';
+import MenuItemPropsTable from 'terra-menu/src/MenuItem?dev-site-props-table';
+import MenuItemGroupPropsTable from 'terra-menu/src/MenuItemGroup?dev-site-props-table';
+
+import ArrowMenu from '../example/ArrowMenu?dev-site-example';
+import BasicMenu from '../example/BasicMenu?dev-site-example';
+import MenuBounded from '../example/MenuBounded?dev-site-example';
+import MenuWidths from '../example/MenuWidths?dev-site-example';
+import MenuWithInstructionsForUse from '../example/MenuWithInstructionsForUse?dev-site-example';
+
+# Terra Menu
+
+The menu is a popup component that displays a list of items, item groups, and dividers. Menu Items can be actionable, have toggle-style selection, or have nested submenu items. Menu Item groups are a single-select grouping of menu items.
+The Menu will determine the height of the popup based on the number of items in the main menu.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-menu`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import Menu from 'terra-menu';
+```
+
+[View Component Source Code](https://github.com/cerner/terra-framework/tree/master/packages/terra-menu)
+
+## Examples 
+<BasicMenu title='Default Menu' />
+<ArrowMenu title='Menu With Arrow' />
+<MenuBounded title='Menu Bounded' />
+<MenuWidths title='Menu Widths' />
+<MenuWithInstructionsForUse title='Help Menu Item: Electronic Instructions For Use' description="The Electronic Instructions For Use (eIFU) icon is a regulatory requirement for CE Mark Certification and Compliance, and is used to indicate Help content that is the equivalent of a manufacturer\'s instruction manual. The label for the Help menu option should be the name of the application followed by 'Help.'"  />
```

Comment:
Can you break this into new lines for readability?

Reply:
Fixed 7dfc345